stack-9.0.1.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: nightly-2021-09-16
+resolver: nightly-2021-11-06
```

Comment:
I think this `package` should also be fixed.

Reply:
I am afraid we cant uncomment packages here cause stack tries to build them even if hls has those plugins disabled.
cabal-install takes them in account for the build plan (so i had to add some allo-newer) but only builds them if they are enabled in hls